wdfn-server/waterdata/templates/partials/monitoring_location_header.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'partials/header.html' %}
+
+{% block page_specific_menus %}
+    {% if stations|length == 1 %}
+        <li class="usa-nav__primary-item">
+            <button class="usa-accordion__button usa-nav__link"
+                    aria-expanded="false"
+                    aria-controls="inventory-section">
+                <span>Data Inventory</span>
+            </button>
+            <ul id="inventory-section" class="usa-nav__submenu">
+                <li class="usa-nav__submenu-item">
+                    <a href="https://waterdata.usgs.gov/nwis/inventory/?site_no={{ stations[0].site_no }}&agency_cd={{ stations[0].agency_cd }}"
+                        target="_blank">
```

Comment:
Did we ant to add the 'rel="noopener"' to these links? I don't think there is a big risk here since they all go to pretty trusted URLs, but we have used it in similar situations elsewhere